data-prepper-api/src/test/java/org/opensearch/dataprepper/model/source/coordinator/SourcePartitionStatusTest.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.model.source.coordinator;
+
+import org.junit.Test;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+public class SourcePartitionStatusTest {
+
+    @Test
+    public void testSourcePartitionStatus() {
+        assertThat(SourcePartitionStatus.ASSIGNED.name(), equalTo(SourcePartitionStatus.ASSIGNED.name()));
```

Comment:
This appears to be a tautology. Why is this test here?

Reply:
Jacoco code coverage has to be 100%. I'll remove this once these are actually used in the future.

Reply:
Thanks for explaining. This works for now.